litellm/integrations/literal_ai.py

```diff
@@ -0,0 +1,193 @@
+#### What this does ####
+# This file contains the LiteralAILogger class which is used to log steps to the LiteralAI observability platform.
+import copy
+import traceback
+
+
+import litellm
+from litellm._logging import verbose_logger
+from litellm.litellm_core_utils.redact_messages import redact_user_api_key_info
+
+
+class LiteralAILogger:
```

Comment:
@willydouhard could you please use the custombatchlogger here? 

https://github.com/BerriAI/litellm/blob/ab153c3e8dab6be093eba3726ada19da176ee8cc/litellm/integrations/langsmith.py#L66

it would prevent issues during high traffic